source/_patterns/basics/forms/checkbox.hbs

```diff
@@ -0,0 +1,11 @@
+{{!--
+formCheckboxInline - set it to true to create inline checkboxes
```

Comment:
If Patternfly only recommends inline checkboxes, should we make inline the default? We can use #unless to remove the class.
(But I'm not positive whether inline is the only/preference/just an option)